components/butterfly/src/server/inbound.rs

```diff
@@ -222,6 +226,6 @@ fn process_ping_mlw(server: &Server, socket: &UdpSocket, addr: SocketAddr, mut m
         server.insert_member_mlw(msg.from, Health::Alive);
     }
     for membership in msg.membership {
-        server.insert_member_from_rumor_mlw(membership.member, membership.health);
+        server.insert_member_from_rumor_mlw_smw(membership.member, membership.health);
```

Comment:
The function we're in (`process_ping_mlw`) needs to have the `_smw` noted in its name and doc comment (and similarly with its callers).

Reply:
Thanks for catching this.  I'll get it updated.